datasets/epa-ch4emission-v2express.data.mdx

```diff
@@ -17,10 +17,19 @@ taxonomy:
   - name: Product Type
     values:
       - Gridded Inventory
+
+infoDescription: | 
+  ::markdown
+      - **Temporal Extent:** 2012 - 2020    
```

Comment:
I think there is one too many indents here, the list is showing up incorrectly in the browser
![image](https://github.com/user-attachments/assets/72657116-2314-4576-9b15-012b2c57db03)

Reply:
<img width="1250" alt="image" src="https://github.com/user-attachments/assets/b2b6bfc9-bc27-46aa-b305-7fa139fdd07a">

Good catch, I thought I looked at all of these... i think it's fixed now